webclient/src/websocket/commands/SessionCommands.ts

```diff
@@ -132,35 +136,35 @@ export class SessionCommands {
 
     webClient.protobuf.sendSessionCommand(sc, raw => {
       switch (raw.responseCode) {
-        case webClient.protobuf.controller.Response.ResponseCode.RespOk:
+        case webClient.protobuf.controller.Response.ResponseCode.RespOk: {
           const passwordSalt = raw['.Response_PasswordSalt.ext'].passwordSalt;
           SessionCommands.login(passwordSalt);
           break;
-
-        case webClient.protobuf.controller.Response.ResponseCode.RespRegistrationRequired:
+        }
+        case webClient.protobuf.controller.Response.ResponseCode.RespRegistrationRequired: {
           SessionCommands.updateStatus(StatusEnum.DISCONNECTED, 'Login failed: incorrect username or password');
           SessionCommands.disconnect();
           break;
-
-        default:
+        }
+        default: {
           SessionCommands.updateStatus(StatusEnum.DISCONNECTED, 'Login failed: Unknown Reason');
           SessionCommands.disconnect();
-          break;
+        }
       }
     });
   }
 
   static register(): void {
-    const options = webClient.options as unknown as ServerRegisterParams;
+    const { userName, password, email, country, realName } = webClient.options as unknown as ServerRegisterParams;
 
     const registerConfig = {
       ...webClient.clientConfig,
-      userName: options.user,
-      password: options.pass,
-      email: options.email,
-      country: options.country,
-      realName: options.realName,
-      clientid: 'webatrice'
+      clientid: guid(),
```

Comment:
Might want to leave some kind of bread crumbs this GUID isn't real, having webatrice in it seemed useful (same for other areas of change)